### PR TITLE
InteractItemframes Flag and Command

### LINKED ISSUE #16: [Flag] interact Itemframe
When not enabled, this will prevent players from the following

- adding items into item frames
- removing items from item frames
- rotating items in item frames

### LINKED ISSUE #67: [Feature] General plugin info command
# Feature request: Misc

## Description of feature?

A command for getting general information about the plugin, such as plugin version. 

The following information should be sent:

```txt
Name:
Version:
Github: 
Zones: <Count>
```

## Extra info

``/zone info``

### PR DESCRIPTION
**Pull request by Lukamon24**
Done #16

### CODE REVIEW

File: src/main/java/org/zone/commands/structure/region/flags/interact/itemframe/ZoneFlagInteractItemframesEnableDisableCommand.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+`* Used to enable/disable the flag {@link InteractItemframesFlag}
+ */
+
+public class ZoneFlagInteractItemframesEnableDisableCommand implements ArgumentCommand {
+
+    public static final ExactArgument REGION = new ExactArgument("region");
+    public static final ExactArgument FLAG = new ExactArgument("flag");
+    public static final ZoneArgument ZONE_VALUE = new ZoneArgument("zone_value", new ZoneArgument.ZoneArgumentPropertiesBuilder());
+    public static final ExactArgument INTERACT_ITEMFRAMES = new ExactArgument("interacyitemframes");
+    public static final BooleanArgument ENABLEDISABLE = new BooleanArgument("enableValue", "enable", "disable");
```
Comment (mosemister) [inline src/main/java/org/zone/commands/structure/region/flags/interact/itemframe/ZoneFlagInteractItemframesEnableDisableCommand.java:32]: interacty ItemFrame?

Comment (Lukamon24) [reply src/main/java/org/zone/commands/structure/region/flags/interact/itemframe/ZoneFlagInteractItemframesEnableDisableCommand.java:32]: XD sorry keyboard problems

**Review by mosemister — CHANGES_REQUESTED** [review]
Just one argument id that im questioning but good otgerwise

**Comment by Lukamon24** [comment]
done dude

File: src/main/java/org/zone/commands/structure/region/flags/interact/itemframe/ZoneFlagInteractItemframesViewCommand.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+/**
+ * Used to view the details of {@link InteractItemframesFlag}
+ */
+public class ZoneFlagInteractItemframesViewCommand implements ArgumentCommand {
+
+    public static final ExactArgument REGION = new ExactArgument("region");
+    public static final ExactArgument FLAG = new ExactArgument("flag");
+    public static final ZoneArgument ZONE_VALUE = new ZoneArgument("zone_value", new ZoneArgument
+            .ZoneArgumentPropertiesBuilder()
+            .setLevel(GroupKeys.INTERACT_ITEMFRAME));
```
Comment (mosemister) [inline src/main/java/org/zone/commands/structure/region/flags/interact/itemframe/ZoneFlagInteractItemframesViewCommand.java]: This should not check the level. If it doesnt have the flag then it should say that

**Review by mosemister — CHANGES_REQUESTED** [review]
One tiny change

**Review by mosemister — APPROVED** [review]
Approved

**Pull request merged**